Task3.py

```diff
@@ -0,0 +1,26 @@
+# Task3! Sorting Dictionary by Key and Value
+def sort_by_key(input_dic_):
+    return dict(sorted(input_dic_.items(), key=lambda item: item[0]))
+
+
+def sort_by_value(input_dic_):
+    return dict(sorted(input_dic_.items(), key=lambda item: int(item[1])))
```

Comment:
No need to cast into int()

Reply:
well i sorted them withe their int value instead of numeric comparison. I'll change it